config/public-api-v1.yml

```diff
@@ -31,31 +31,365 @@ paths:
-            - attributes
-          properties:
-            id:
-              type: integer
-            type:
-              type: string
-            attributes:
-              $ref: "#/components/schemas/Course"
-    Course:
+          $ref: "#/components/schemas/CourseData"
+        # included:
+        #   $ref: "#/components/schemas/Included"
+        jsonapi:
+          $ref: "#/components/schemas/JSONAPI"
+    CourseAttributes:
       type: object
       required:
         - code
       properties:
+        about_accredited_body:
+          type: string
+          format: markdown
+          description: Description of the accredited body for this course.
+          example: ""
+        about_course:
+          type: string
+          description: Short factual summary of the course formatted as Markdown.
+          example: |
+            The Art and Design PGCE is a challenging and forward-looking
+            programme which prepares students to teach across the 11-16 age
+            range, encouraging them to relate art, craft and design education to
+            contemporary art practice. It has a strong reputation for promoting
+            innovation in education theory, practice and policy. This programme
+            aims to inform and inspire, to challenge orthodoxies and encourage a
+            freshness of vision. It provides support and guidance for learning
+            and teaching in art and design, identifying strategies to motivate
+            and engage pupils in making, discussing and evaluating visual and
+            material culture. The IOE provides excellent studio space and
+            facilities for Art and Design, including a computing suite where
+            students can learn how technology is used in art and design
+            education. The programme also has strong links with galleries,
+            museums and other sites for learning, which are recognised as an
+            important resource for engaging students in cultural and social
+            issues.Through seminars and studio-based activities, students will
+            study the concepts, processes and skills of art, craft and design,
+            sharing their knowledge and understanding with other student
+            teachers and consider how it relates to the secondary curriculum.
+            Towards the end of the PGCE, students will build on their own
+            practice by initiating a curriculum development project, culminating
+            in the display of their own work in a final exhibition that
+            represents their personal philosophy for art and design education.
+        accredited_body_code:
+          type: string
+          description: |
+            Unique provider-code for the accredited body of this course. Only
+            present if the course is accredited by a provider other than the one
+            running this course.
+          maxLength: 3
+          example: 2FR
+        age_range_in_years:
```

Comment:
Can we move away from this format now? I know we're still storing it like that internally but would `minimum_age` `maximum_age` start moving us away from using the string?

Reply:
That's a good point. Our data looks like:

```
> puts rc.courses.pluck(:age_range_in_years).uniq.compact.sort
0_to_5
11_to_16
11_to_18
11_to_19
14_to_18
14_to_19
16_to_18
16_to_19
2_to_19
3_to_11
3_to_7
3_to_9
4_to_11
5_to_11
7_to_11
7_to_14
```

I like the idea, will have a think.

Reply:
How about `age_minimum` and `age_maximum`?

Reply:
That would definitely be an improvement.

Reply:
:+1: to this - we allow custom age ranges so I think it should be separate on the public API